osu.Game/Graphics/UserInterface/HoverClickSounds.cs

```diff
@@ -16,16 +18,38 @@ namespace osu.Game.Graphics.UserInterface
     public class HoverClickSounds : HoverSounds
     {
         private SampleChannel sampleClick;
+        private readonly MouseButton[] buttons;
 
+        /// <summary>
+        /// Creates an instance that adds sounds on hover and left click only.
+        /// </summary>
+        /// <param name="sampleSet">Set of click samples to play.</param>
         public HoverClickSounds(HoverSampleSet sampleSet = HoverSampleSet.Normal)
+            : this(new[] { MouseButton.Left }, sampleSet)
+        {
+        }
+
+        /// <summary>
+        /// Creates an instance that adds sounds on hover and on click for any of the buttons specified.
+        /// </summary>
+        /// <param name="buttons">Array of button codes which should trigger the click sound.</param>
+        /// <param name="sampleSet">Set of click samples to play.</param>
+        public HoverClickSounds(MouseButton[] buttons, HoverSampleSet sampleSet = HoverSampleSet.Normal)
```

Comment:
```suggestion
        public HoverClickSounds(MouseButton[] buttons = new[] { MouseButton.Left }, HoverSampleSet sampleSet = HoverSampleSet.Normal)
```

Reply:
This won't actually compile, a default value parameter must be a compile-time constant. Best I could do here is default to `null` and then coalesce:
```csharp
public HoverClickSounds(MouseButton[] buttons = null, HoverSampleSet sampleSet = HoverSampleSet.Normal)
    : base(sampleSet)
{
    this.buttons = buttons ?? new[] { MouseButton.Left };
}
```

Reply:
Better than another constructor.